kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/archetype-resources/src/main/java/__artifactId__/DefaultController.java

```diff
@@ -0,0 +1,19 @@
+package ${groupId}.${artifactId};
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class DefaultController {
```

Comment:
do we really need this? Can't we just have static html page that will be filtered on project creation and then we don't need this at all.